### PR TITLE
docs: fix getting-started.md typo

### PR DESCRIPTION
**Pull request by ppborah**
**What this PR does / why we need it**: This PR fixes typos in docs (getting-started.md): Highlighted in red 👇
fix getting-started md typo
�

### CODE REVIEW

**Comment by ppborah** [comment]
Please review :)

File: docs/content/getting-started.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ title: Getting started
 
 # Getting started
 
-HyperShift is middleware for hosting OpenShift control planes at scale that
```
Comment (csrwng) [inline docs/content/getting-started.md:7]: I think the original wording is more correct

File: docs/content/getting-started.md [hunk 2]
```diff
@@ -4,8 +4,8 @@ title: Getting started
...
-solves for cost and time to provision, as well as portability cross cloud with
```
Comment (csrwng) [inline docs/content/getting-started.md:8]: s/across cloud/across clouds/

**Review by csrwng — COMMENTED** [review]
Thanks, a couple of comments.

**Comment by enxebre** [comment]
@ppborah can you address https://github.com/openshift/hypershift/pull/1534#discussion_r978754988 and then we merge? thanks!

**Comment by ppborah** [comment]
@enxebre I have addressed it, could you please take a look ? Thanks :)
Screenshot 2022-10-05 at 3 02 27 PM

File: docs/content/getting-started.md (hunk 2 above)
Comment (ppborah) [reply docs/content/getting-started.md:8]: I think we should use `cloud service providers` instead of `cloud providers`

**Pull request merged**